dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -748,6 +749,20 @@ export const LiveLayout = ({ CAPI, NAV, format, palette }: Props) => {
 										/>
 									</div>
 								</GridItem>
+								<GridItem area="matchstats">
+									{CAPI.matchUrl && (
```

Comment:
Out of curiosity, in StandardLayout we check both `matchUrl` and the design format (`ArticleDesign.MatchReport`) before rendering any match stats related stuff. For live/deadblogs should we display match stats for any blog that specifies a matchUrl, regardless of the article format?

Reply:
As liveblogs currently only have one layout expected. i.e. there's no showcase/immersive main media or other elements, we don't need to worry about the article format. That's my understanding at least.